src/core/types.h

```diff
@@ -147,6 +148,9 @@ struct ParamNames {
     std::vector<llvm::StringRef> args;
     llvm::StringRef vararg, kwarg;
 
+    std::vector<AST_Name*> arg_names;
+    AST_Name* vararg_name, *kwarg_name;
```

Comment:
Could we get rid of the StringRef versions?  or at least have a note about when one should be used vs the other (I assume that the stringref ones are now deprecated but it'd be nice to have a comment).